versioned_docs/version-v2.18.x/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,680 @@
-# Enabling AT-TLS across your Zowe environment
+<!-- omit in toc -->
+# Configuring AT-TLS for Zowe Server
 
-The communication server on z/OS provides functionality to encrypt HTTP communication for on-platform jobs. This functionality is referred to as Application Transparent Transport Layer Security (AT-TLS).
```

Comment:
Can you validate if this introduction is accurate?

You can configure parameters in the Zowe server to enable Zowe work with AT-TLS. Review this article for information about AT-TLS inbound and outbound rules, and the required configuration to use AT-TLS in high availability. You can also find troubleshooting tips as well as security recommendations.